base/templates/base/home.html

```diff
@@ -0,0 +1,14 @@
+{% extends 'base.html' %}
+
+{% block content %}
+
+<h4>Welcome {{request.user.Username}}</h4>
```

Comment:
This content is not up to date with the remote repo. Consider git pull once you're on your working branch.
make sure this file is up to date with the remote one. You can refer with remote main branch